### PR TITLE
Add EOL notice

### PR DESCRIPTION
**Pull request by Grumbel**

### CODE REVIEW

**Comment by AsciiWolf** [comment]
jstest-gtk looks like a really useful tool. It would be pity if it was removed from Flathub. I can take over the Flatpak if needed.

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "end-of-life": "This application is now being maintained as a Nix flake."
```
Comment (AsciiWolf) [inline flathub.json:2]: By the way, this information is not useful for most Flatpak users since most of them do not use NixOS.

Comment (Grumbel) [reply flathub.json:2]: You don't need to run NixOS to run Nix flakes, they work everywhere.

Comment (AsciiWolf) [reply flathub.json:2]: Ah, I did not know that.

**Comment by AsciiWolf** [comment]
@hfiguiere @barthalion Could you please revert the EOL commit and set me as a new maintainer? Thanks!

**Comment by hfiguiere** [comment]
I can't set anyone as maintainer. I haven't been granted these powers.

**Comment by AsciiWolf** [comment]
Ah. So let's wait for @barthalion. :-)

**Comment by AsciiWolf** [comment]
> Update for 2018: The /dev/input/jsX interface that jstest-gtk uses is largely obsolete. Most newer software will use /dev/input/eventX, so neither the calibration nor the remapping functionality will have any effect.

(Source: https://jstest-gtk.gitlab.io)

If this is true, I am probably interested in [evtest-qt](https://github.com/flathub/io.gitlab.evtest_qt.evtest_qt) instead of jstest-gtk.

**edit:** But if [this](https://github.com/Grumbel/evtest-qt#usage) is also true, it is probably pointless to maintain any of these two packages.

**Comment by AsciiWolf** [comment]
I have changed my mind about the jstest-gtk and evtest-qt Flatpaks. If they do not work properly (as stated in my previous comment), there is no point in keeping them on Flathub. Also, the upstream [removal](https://gitlab.com/jstest-gtk/jstest-gtk/-/commit/72c5ee0e2b3f8858889370b5666e62e256e361d3) of AppStream metadata indicates that these projects are probably not intended to be maintained for modern Linux distributions anymore.

**Pull request merged**